electron/ipc/handlers.ts

```
@@ -949,4 +951,249 @@ export function registerIpcHandlers(
+					(resolve) => {
+						ffmpegProcess.on("close", (code, signal) => {
+							resolve({ code, signal });
+						});
+						ffmpegProcess.on("error", (err) => {
+							console.error("[FFmpeg] Process error:", err);
+							resolve({ code: -1, signal: null });
+						});
+					},
+				);
+
+				let stderrOutput = "";
```

Biome (2.4.10)

[error] This variable stderrOutput is unused.

(lint/correctness/noUnusedVariables)